cinema_service/settings.py

```diff
@@ -137,3 +137,10 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+
+REST_FRAMEWORK = {
+   "DEFAULT_PERMISSION_CLASSES": [
+       "rest_framework.permissions.IsAuthenticated",
```

Comment:
The `DEFAULT_PERMISSION_CLASSES` is set to `IsAuthenticated`. This means all views will require authentication by default. Ensure this aligns with your application's requirements, as it might restrict access to unauthenticated users.